backend/services/seeder.py

```
@@ -1,457 +1,508 @@
+        format_engine=wc.get("format_engine", "group_knockout"),
+        odds_api_sport_key=wc.get("odds_api_sport_key", "soccer_fifa_world_cup"),
+        home_advantage_elo=0,
+        neutral_venue=True,
+    )
+    tourney = _get_or_create_tournament(db, comp, wc.get("season", "2026"))
+
+    try:
+        print("Fetching live Elo ratings from eloratings.net...")
+        live_elo = fetch_current_elo_ratings()
+        print(f"Successfully fetched {len(live_elo)} Elo ratings from eloratings.net.")
+    except Exception as exc:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)